src/components/pages/gallery/Upload.tsx

```diff
@@ -261,6 +322,8 @@ export default function Upload(props: Props) {
             appContext.resetSharedFiles();
             props.setUploadInProgress(false);
             props.syncWithRemote();
+            if (isPendingDesktopUpload.current)
```

Comment:
It wouldn't have worked that way, since we wanted this value to be persisted till the upload is over as it is used in the cancel uploads button, but since we are now making the button visible for all uploads I think it doesn't matter now.

Reply:
cools then we can ignore this

Reply:
I have reverted back this change.